ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml

```diff
@@ -172,7 +172,7 @@ tests:
         NETWORK_TYPE=OVNKubernetes
         OVN_LOCAL_GATEWAY_MODE=true
     workflow: baremetalds-e2e
-- as: e2e-metal-ipi-compact
+- as: e2e-metal-ipi-compact-ipv6
```

Comment:
Is it intentional or should be `ovn-ipv6`?

Reply:
thanks not intentional, will fix